docs/data.json

```diff
@@ -2,9 +2,16 @@
     "project": {
         "name": "c2fo-ui-components",
         "description": "C2FO core ui components",
-        "version": "0.0.3"
+        "version": "1.0.0"
```

Comment:
Is this really a `1.0.0` release? It can be, I just want to make sure this is ready for production, which is usually what a `1.0` release signifies.